src/Native/Unix/System.Security.Cryptography.Native/opensslshim.h

```diff
@@ -422,9 +437,11 @@ void SSL_get0_alpn_selected(const SSL* ssl, const unsigned char** protocol, unsi
     REQUIRED_FUNCTION(RSA_size) \
     REQUIRED_FUNCTION(RSA_up_ref) \
     REQUIRED_FUNCTION(RSA_verify) \
-    REQUIRED_FUNCTION(SSL_CIPHER_description) \
+    LIGHTUP_FUNCTION(SSL_CIPHER_find) \
```

Comment:
You defined it here as LIGHTUP (it's allowed to be missing) but then defined a local_ for it (which provides the binding for FALLBACK.

Which state are you intending?

Reply:
seems that changing to FALLBACK broke the build, will debug tomorrow

Reply:
I think I actually want LIGHTUP since we can't really work without OpenSSL 1.1.1 (SSL_CIPHER_find missing)